tests/MessageUnitTest.php

```diff
@@ -2,6 +2,7 @@
 
 use Razorpay\Slack\Message;
 use Razorpay\Slack\Attachment;
+use PHPUnit\Framework\TestCase as PHPUnit_Framework_TestCase;
```

Comment:
Is this needed?

Reply:
yes, PHPUnit_Framework_TestCase is not present directly